plugins/modules/vmware_dvs_portgroup.py

```diff
@@ -17,205 +18,231 @@
+                default: false
+            port_config_reset_at_disconnect:
+                type: bool
+                description: Indicates if the configuration of a port is reset automatically after disconnect.
+                default: True
             security_override:
-                description:
-                - Indicates if the security policy can be changed per port.
-                default: False
                 type: bool
-            shaping_override:
-                description:
-                - Indicates if the shaping policy can be changed per port.
+                description: Indicates if the security policy can be changed per port.
                 default: False
+            shaping_override:
                 type: bool
-            traffic_filter_override:
-                description:
-                - Indicates if the traffic filter can be changed per port.
+                description: Indicates if the shaping policy can be changed per port.
                 default: False
+            traffic_filter_override:
                 type: bool
-            uplink_teaming_override:
-                description:
-                - Indicates if the uplink teaming policy can be changed per port.
+                description: Indicates if the traffic filter can be changed per port.
                 default: False
+            uplink_teaming_override:
                 type: bool
+                description: Indicates if the uplink teaming policy can be changed per port.
+                default: False
             vendor_config_override:
-                description:
-                - Indicates if the vendor config can be changed per port.
                 type: bool
+                description: Indicates if the vendor config can be changed per port.
                 default: False
             vlan_override:
-                description:
-                - Indicates if the vlan can be changed per port.
                 type: bool
-                default: False
+                description: Indicates if the vlan can be changed per port.
+                default: false
+        required: False
         default: {
             'traffic_filter_override': False,
             'network_rp_override': False,
-            'live_port_move': False,
             'security_override': False,
             'vendor_config_override': False,
             'port_config_reset_at_disconnect': True,
             'uplink_teaming_override': False,
             'block_override': True,
             'shaping_override': False,
             'vlan_override': False,
-            'ipfix_override': False
+            'netflow_override': False
         }
-        type: dict
+        aliases: ['port_policy']
+    netflow_enabled:
+        description:
+            - Indicates if NetFlow is enabled on the uplink portgroup.
+        type: bool
+        default: False
```

Comment:
Please add `version_added: '1.5.0'` if the option is new.

Reply:
done